docs/features/sessions/6_machines.md

```diff
@@ -0,0 +1,84 @@
+---
+sidebar_position: 1.6
+title: Machines
+sidebar_label: Machines
+---
+
+# Machines
+
```

Comment:
I'd prefer this just be a link. Like "Please see our [machines page](https://docs.grid.ai/platform/machines) to see all the machine types available to you and the proper syntax to employ them". 

The link currently won't work since it isn't live and only resides on the dev branch now.